docs/json/radarr/cf/anime-bd-tier-02-seadex-muxers.json

```diff
@@ -114,6 +114,15 @@
         "value": "\\[Crow\\]|-Crow\\b"
       }
     },
+    {
+      "name": "CsS",
+      "implementation": "ReleaseTitleSpecification",
+      "negate": false,
+      "required": false,
+      "fields": {
+        "value": "\\b(CsS)\\b"
```

Comment:
Typo this should be Cunnysseur yes?

Reply:
Actually Cunnysseur is added below it, I moved CsS above CUNNY because the order was off in the original file which is why it is showing up here

Reply:
Ah my bad I see. Thanks